src/component/form.js

```diff
@@ -1,11 +1,20 @@
+import { useDispatch } from 'react-redux/es/exports';
+import React, { useRef } from 'react';
+import uniqid from 'uniqid';
+import { addBook } from '../redux/books/books';
 import './books.css';
 
-const Form = () => (
-  <form className="forms">
-    <input type="text" name="author" placeholder="Author" />
-    <input type="text" name="book" placeholder="Book" />
-    <input type="submit" value="Submit" />
-  </form>
-);
+const Form = () => {
+  const dispatch = useDispatch();
+  const title = useRef(null);
+  const author = useRef(null);
+  return (
+    <form className="forms" onSubmit={(e) => { e.preventDefault(); dispatch(addBook(title.current.value, author.current.value, uniqid())); e.target.reset(); }}>
+      <input ref={title} type="text" name="author" placeholder="Author" />
+      <input ref={author} type="text" name="book" placeholder="Book" />
+      <button type="submit">Add</button>
+    </form>
+  );
+};
```

Comment:
- Hey, please kindly ensure your form doesn't get submitted with an empty field. Please, kindly fix this issue. You can move the logic here in another function, and do some check to ensure the inputs fields are filled before the form get submitted